kitsune/kbforums/models.py

```diff
@@ -51,7 +51,7 @@ class Meta:
     @property
     def last_page(self):
         """Returns the page number for the last post."""
-        return self.replies / kbforums.POSTS_PER_PAGE + 1
+        return self.replies // kbforums.POSTS_PER_PAGE + 1
 
     def __unicode__(self):
```

Comment:
Not relevant to this fix. Mainly a note that we need to update these methods too.